### DISCUSSION TITLE
MacOS Ventura

### DISCUSSION THREAD

**Discussion by TwEnTy90** [hackintosh-club/intel-nuc10#74] [Q&A]

**Comment by ruthram25** [comment]
Pleaase find the EFI attached.
[EFI.zip](https://github.com/hackintosh-efi/intel-nuc10/files/9226686/EFI.zip)
.

**Comment by TwEnTy90** [comment]


**Reply by ruthram25** [reply]
Have you unlocked your CFGLOCK? try resetting  NVRAM.. HDMI audio not working.. You can use OC configurator to set the theme and update your SMBIOS .. Apple store is working fine.
Since its an initial commit, there are issues.

**Comment by zearp** [comment]
You can copy the serials/etc to a new config to keep iServices working like I told you in the issue you opened on my repo. But instead of working with me you wanted me to modify your EFI and do the work for you. Then you got upset when I said no. You called me names, spammed my repo with profanity and caps lock filled issues and acted like an entitled child. You told me you have a NUC8. Using a NUC10 config and APCI patches on your NUC8 is not recommended.

> Hi, my names is Zearp i need a suggestion for install Mac Os Ventura. I'm Stuck on boot of USB installer. I use your EFI for the booting.

Why do you pretend to be me? Are you like 12 years old or something? Smh.

**Reply by zearp** [reply]
> Hi, this is not your natural habitat, go don't help your colleagues who have your shitty nuc, thanks

Nice mature reply. You are breaking the terms of service because Github requires you to be 13 or older to sign up.

**Reply by zearp** [reply]
You’re full of it. You didn’t address a single thing I posted. Then you start cursing again and lie about me. You’re one of the people who will cause me to keep my next efi private. Not that you care. You don’t cause you’re a selfish cunt.

**Reply by zearp** [reply]
@essereio why help this guy who pretends to be me, spams and curses and is just a bad person? Look at how he talks to me and lies about me. Smh.

**Comment by TwEnTy90** [comment]
Close the thread unfortunately it is useless to ask for help. Especially if you are insulted by third parties who are pundits or think they are psychologists but in reality they are empty people. As for Zearp, unfortunately I feel sorry for him as a person, not only does he respond badly to people, but if they reply against him he plays the victim and cries. Zearp I apologize if I offended your bad soul by writing jokingly, obviously as a joke, on a post. Maybe now you sleep at night. As for "ESSEREIO" I have to tell you, study psychology better. Maybe you find that it's your passion;) good life phenomena 

@zearp @essereio

**Reply by TwEnTy90** [reply]
> > @essereioThe thing that makes me laugh the most is that you don't know the story and you judge from the outside, as you can see I opened a post in this thread and kindly asked for information. I responded in kind to Zearp as he came under this post to respond. I'm not saying I didn't provoke it, but your answers are inappropriate, especially since I've never disrespected you, or am I wrong? Are you a psychologist or a technology enthusiast? stay in your asshole.
> 
> First you attack and now you play victim. https://psychcentral.com/disorders/narcissistic-personality-disorder/narcissist-plays-the-victim

But is that what Zearp did in the beginning and then I was attacked? Right? @Zearp there is a link for you read it, your friend cares about you

**Reply by TwEnTy90** [reply]
> > @essereioThe thing that makes me laugh the most is that you don't know the story and you judge from the outside, as you can see I opened a post in this thread and kindly asked for information. I responded in kind to Zearp as he came under this post to respond. I'm not saying I didn't provoke it, but your answers are inappropriate, especially since I've never disrespected you, or am I wrong? Are you a psychologist or a technology enthusiast? stay in your asshole.
> 
> First you attack and now you play victim. https://psychcentral.com/disorders/narcissistic-personality-disorder/narcissist-plays-the-victim

Before judging other people, look at yourself. Is it especially if you don't know what happened? Why comments? Study psychology, but as advice

**Reply by TwEnTy90** [reply]
can you show me your degree in psychology? @essereio

**Reply by TwEnTy90** [reply]
that said, anything you say will have no effect. First of all, I repeat you do not know in the least what happened and you speak. If you want, we can clarify once and for all with @zearp "and return friends as before" That's right so you realize you are the champion of this fucking @essere 

I don't play with fire my friend

**Reply by TwEnTy90** [reply]
in this scenario, I am defending myself from an episode that happened with Zearp and you are bullying me, but since I am not afraid of bullies, let's talk about it, so maybe you wake up

**Reply by TwEnTy90** [reply]
Maybe talking about it, since you took the liberty of judging me, show me your degree in Psychology

**Reply by TwEnTy90** [reply]
words have weight. Reporting facts that offend other people is discrimination. Defending a person who has harmed me (no matter how insignificant it may be) and spitting out judgments without having any knowledge of what you are saying is discrimination. You are a bully accept it.

@essereio 

And trying to defend yourself against people like you is not being a victim. My word is as good as yours. Do you want to talk again? Or can you just put random links on topics that you don't even know how heavy they are on people? BULLY.

**Reply by TwEnTy90** [reply]
And I know very well that in the first place I have behaved badly with Zearp. Reacting wrong to how he treated me. But it's one thing to joke about writing the name Zearp. It is one thing to take the defense of a person, without knowing the story and spitting out sentences to zero. This is being bully my friend.

**Reply by TwEnTy90** [reply]
well come on, I see that we have not made progress, I try to defend myself and you send links on topics that you don't even know. Good job. I'm glad you're on fire.

**Reply by TwEnTy90** [reply]
Come on, if you fail like Hackintosher, you can always be a psychologist, ah no, sorry you just send links ...

**Reply by TwEnTy90** [reply]
Hi Bully, say hello to Zearp
